internal/scorecard/plugins/olm_tests.go

```diff
@@ -165,16 +190,49 @@ func NewOLMTestSuite(conf OLMTestConfig) *schelpers.TestSuite {
 		"Test suite checks if an operator's CSV follows best practices",
 	)
 
+	ts.AddTest(NewBundleValidationTest(conf), 1)
 	ts.AddTest(NewCRDsHaveValidationTest(conf), 1.25)
 	ts.AddTest(NewCRDsHaveResourcesTest(conf), 1)
-	ts.AddTest(NewAnnotationsContainExamplesTest(conf), 1)
+	ts.AddTest(NewCSVValidationTest(conf), 1)
 	ts.AddTest(NewSpecDescriptorsTest(conf), 1)
 	ts.AddTest(NewStatusDescriptorsTest(conf), 1)
 
 	return ts
 }
 
-// Test Implentations
+// Test Implementations
+
+// Run - implements Test interface
+func (t *BundleValidationTest) Run(ctx context.Context) *schelpers.TestResult {
+	res := &schelpers.TestResult{Test: t, MaximumPoints: 1}
+
+	if t.OLMTestConfig.Bundle == "" {
+		res.Errors = append(res.Errors, errors.New("add a 'bundle' directory which is required for this test"))
+		return res
+	}
+
+	validationLogOutput := new(bytes.Buffer)
+	logrus.SetOutput(validationLogOutput)
```

Comment:
Include a comment here about why we're doing this.  Also, is it possible to get logrus's current output writer before we set it so that we can put its original writer back when we're finished with this test?

Something like:
```go
origOutput := logrus.GetOutput()
logrus.SetOutput(validationLogOutput)
defer logrus.SetOutput(origOutput)
```

Reply:
I checked with the CVP folks, they have not migrated yet, they were waiting on the fix for https://github.com/operator-framework/operator-sdk/issues/2271 which was holding them up.

Reply:
I have made the change, deferring the reinstatement of the logrus io.Writer from before.